commitlint.config.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
```

Comment:
These seem to be a lot more comprehensive than what we have on other repos - was this intended?

Reply:
@tiagolobocastro Yes, this was intentional.  The deviations in rule checking between this configuration and that currently in place for other repos are relatively few; the principle change is that lower-case is enforced as a formatting/visual styling standard.  If this version proves acceptable in service for this repo, then the plan is to update the other Mayastor repo linters to use the same linting convention.